apps/workflows/apps.py

```diff
@@ -0,0 +1,14 @@
+"""
+
+    Workflows - Issue Workflow Manager
```

Comment:
This is kinda nitpick but still. I know this naming is a result of the previous **OSIM - Open Security Issue Manager** adjustion. Before there was an abbreviation of the name chosen by Garth Mollett and it was in alignment with OSIDB name.

The think I not happy about :smile: is the word Issue. What does it mean in the context of Workflows? In OSIDB there are no issues. Is it mean for Jira issues? Even if so I would see it not obvious and confusing. My suggestion is just remove that word. The other two makes sense.

The line below is now not really precise. It is Python-like file path so the case must match. You can either fix it or remove it. Up to you.

Reply:
Fixed, thanks!